dubbo-rpc/dubbo-rpc-dubbo/src/main/java/org/apache/dubbo/rpc/protocol/dubbo/DubboCodec.java

```diff
@@ -158,6 +158,11 @@ protected Object decodeBody(Channel channel, InputStream is, byte[] header) thro
                     req.setEvent(true);
                 } else {
                     req = new HeartBeatRequest(id);
```

Comment:
Please help change to `Request` here. There is a mistake when recreating heart beat event.

Reply:
Done